packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -464,7 +465,18 @@ async function handler(
   });
 
   const contactOwnerFromReq = reqBody.teamMemberEmail ?? null;
-  const skipContactOwner = reqBody.skipContactOwner ?? false;
+
+  const isReroutingCase = isRerouting({
+    rescheduleUid: reqBody.rescheduleUid ?? null,
```

Comment:
originalRescheduledBookingUserId not available here(as we haven't fetched the booking from DB yet) We should be fine using reqBody.rescheduleUid also